cluster/vagrant/setup_kubernetes_master.sh

```diff
@@ -50,6 +50,35 @@ else
   kubectl create -f kube-$NETWORK_PROVIDER.yaml
 fi
 
+# Create an ingress-nginx setup
+# See: https://github.com/kubernetes/ingress-nginx/tree/master/deploy
+{
+  curl_yaml_combine() { for URL in $@; do curl -L $URL ; echo --- ; done }
+
+  # Do the basic deployment
+  curl_yaml_combine \
+    https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/default-backend.yaml \
+    https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/configmap.yaml \
+    https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/tcp-services-configmap.yaml \
+    https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/udp-services-configmap.yaml \
+    https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/rbac.yaml \
+      | sed "s#namespace: ingress-nginx#namespace: kube-system#" \
+      | kubectl apply -f -
+
+  # Pin the controller to the master
+  ( curl -L https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/with-rbac.yaml && \
+    echo -e "      nodeSelector:\n        kubernetes.io/hostname: master" \
```

Comment:
Can we export nodeSelector from a config file instead of hardcode it?

Reply:
Actually - I think we can drop this pinning in general